hackathons/usw25: Update sessions and details (#487)
unikraft/docs: content/hackathons/usw25/index.mdx

---

It focuses on cloud-native applications and on the unikernel technology that powers efficient and high performance cloud deployments.

> This is the fifth edition of the event, after USoC'21 to USoC'23, and USW'25, we will focus on building and running applications using Unikraft.

Reviewer: ```suggestion
This is the fifth edition of the event, after USoC'21 to USoC'23, and USW'24, we will focus on building and running applications using Unikraft.
```

The three-week event holds a number of starter tutorials and workshops on how to configure, build, run and debug applications using Unikraft.

There will be 6 sessions taking place in the first two weeks (between June 23 and July 4, 2025).

> Each session takes place for 3 hours, in the 4pm-7pm CEST, in English.

Reviewer: ```suggestion
Each session takes place for 3 hours, in the 4pm-7pm CEST interval, in English.
```

Sessions will consist of talks and demos delivered by members of the Unikraft community, followed by practical tutorials that you will work on with support and supervision.
Sessions take place on [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).

Topics include building unikernels, benchmarking, debugging, porting applications, virtualization and platform specifics.

> The 3 sessions will focus on the internals of Unikraft: the build system, native configuration options, application porting.

Reviewer: ```suggestion
The 6 sessions will focus on the internals of Unikraft: the build system, native configuration options, application porting.
```

### Registration

> If you're eager to learn more about efficient cloud computing and unikernel technology, to work on practical open source tasks and to expand your knowledge of low-level topics, you'll want to be part of USW'25.

Reviewer: ```suggestion
If you're eager to learn more about efficient deployments and unikernel technology, to work on practical open source tasks and to expand your knowledge of low-level topics, you'll want to be part of USW'25.
```
